src/soap_wsse/signing.py

```diff
@@ -131,14 +135,18 @@ def verify_envelope(reply, key_file):
     dsigCtx = xmlsec.DSigCtx()
 
     xmlsec.addIDs(doc, ['Id'])
-    signKey = xmlsec.Key.load(key_file, xmlsec.KeyDataFormatPem)
+    signKey = xmlsec.Key.load(
+        key_file,
+        xmlsec.KeyDataFormatPem,
+        password,
+    )
     signKey.name = os.path.basename(key_file)
 
     dsigCtx.signKey = signKey
     try:
         dsigCtx.verify(node)
-    except xmlsec.VerificationError:
-        return False
+    except xmlsec.VerificationError, e:
+        return e.args[1] == 2
```

Comment:
I am really not sure how to properly fix that. My SOAP response was correct (from my sight) but deemed as invalid. I needed it passed. So, probably there's much better way how to deal with.